README.md

```diff
@@ -1,29 +1,22 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Portfolio
 
-> One paragraph statement about the project.
+> Creating a protfolio to display my current skill level.
 
-![screenshot](./app_screenshot.png)
-
-Additional description about the project and its features.
+Fully functional on Mobile.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML,CSS
+- No Frameworks
+- No Technologies
 
 ## Live Demo
-
-[Live Demo Link](https://livedemo.com)
-
+images/demo.png
```

Comment:
# [OPTIONAL]
- [ ] You don't have a live demo link, consider either deploying the project with GitHub pages or removing this section